br: update docs (#3506)
pingcap/docs: br/backup-and-restore-faq.md

Because these system libraries do not exist in the backup files, no conflict occurs among system tables during data restoration.

> ## How should I handle the `Permission denied` error, even if I have tried to run BR using root in vain?

Reviewer: ```suggestion
## What should I do to handle the `Permission denied` error, even if I have tried to run BR using root in vain?
```

>
> Therefore, It is recommended to check the permission before data restoration.

> ## How should I handle the `Io(Os...)` error?

Reviewer: ```suggestion
## What should I do to handle the `Io(Os...)` error?
```

Almost all of these problems are system call errors that occur when TiKV writes data to the disk. You can check the mounting method and the file system of the backup directory, and try to back up data to another folder or another hard disk.
 
For example, you might encounter the `Code: 22(invalid argument)` error when backing up data to the network disk built by `samba`.

> ## How should I handle the `rpc error: code = Unavailable desc =...` error occurred in BR?

Reviewer: ```suggestion
## What should I do to handle the `rpc error: code = Unavailable desc =...` error occurred in BR?
```

 
This error might occur when the capacity of the cluster to restore (using BR) is insufficient. You can further confirm the cause by checking the monitoring metrics of this cluster or the TiKV log.

> To handle this issue, you can try to scale out the cluster resources, reduce the concurrency during restoration, and enable the `RATE_LIMIT` option.

Reviewer: `restore` is a noun used in our documentation and some other IT companies such as CockroachDB.
```suggestion
To handle this issue, you can try to scale out the cluster resources, reduce the concurrency during restore, and enable the `RATE_LIMIT` option.
```

Author: got it:)

 
For example, you might encounter the `Code: 22(invalid argument)` error when backing up data to the network disk built by `samba`.

> ## What should I do to resolve the `rpc error: code = Unavailable desc =...` error?

Reviewer: ```suggestion
## How should I handle the `rpc error: code = Unavailable desc =...` error occurred in BR?
```

> This problem is generally caused by insufficient performance of the cluster restoration when using BR to restore data. You can further confirm the cause by cluster restoration monitoring or TiKV logs.

Reviewer: ```suggestion
This error might occur when the capacity of the cluster to restore (using BR) is insufficient. You can further confirm the cause by checking the monitoring metrics of this cluster or the TiKV log.
```

> To solve this kind of problem, you can try to expand cluster resources, reduce the concurrency during restoration, and turn on the `RATE_LIMIT` setting.

Reviewer: ```suggestion
To handle this issue, you can try to scale out the cluster resources, reduce the concurrency during restore, and enable the `RATE_LIMIT` option.
```
